src/TransferGroup.cc

```diff
@@ -206,6 +206,29 @@ namespace ChimeraTK {
     // set flag on the accessors that it is now in a transfer group
     accessor.getHighLevelImplElement()->_isInTransferGroup = true;
 
+    // Apply possible replacements by calling replaceTransferElement() in all possible combinations, i.e. pass all
+    // internal elements in this group to replaceTransferElement() called on all high-level elements in the group. Keep
+    // in mind that replaceTransferElement() is supposed to pass the call down to internal implementations, so this will
+    // effectively try all combinations.
+    //
+    // Start with the abstractor, to make sure we replace at the highest level if possible, but only if this isn't a
+    // temporary abstractor. We need to do this in a separate loop first, so we are not using deeper levels inside the
+    // abstractor as a replacement somewhere else after the implementation in the abstractor might have been replaced.
+    if(not isTemporary) {
+      for(const auto& hlElem1 : _highLevelElements) {
+        auto list = hlElem1->getInternalElements();
+        list.push_front(hlElem1);
+
+        for(const auto& replacement : list) {
+          // try on the abstractor first, to make sure we replace at the highest
+          // level if possible, but only if this isn't a temporary abstractor
```

Comment:
This comment somehow feels off

Reply:
yes sorry, it was left in from a previous version